### PR TITLE
Feat: Added landing page

### LINKED ISSUE #7: Update : index.html
Adding links of login, signUp and product listing page in index.html

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. landing page
ShoestoreLandingPage
2. categories list
Categories
3. categories list after hovering effect
categoriesList

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                        <span>g</span>
+                        <span>o</span>
+                        <span>r</span>
+                        <span>i</span>
+                        <span>e</span>
+                        <span>s</span>
+                    </h1>
+                    <div class="box-container">
+                        <div class="box">
+                            <img src="https://images.unsplash.com/photo-1552422554-0d5af0c79fc6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
```
Comment (vivekmehto) [inline index.html]: we should give ``alt`` to every image it helps the screen reader to describe the image.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                        <div class="box">
+                            <img src="https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
+                            <div class="content">
+                                <h1>Sports Shoes</h1>
+                                <p>Shop from a wide range of sports shoes.</p>
+                                <a href="#" class="btn">see More</a>
+                            </div>
+                        </div>
+                        <div class="box">
+                            <img src="https://images.unsplash.com/photo-1481729379561-01e43a3e1ed4?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=970&q=80" alt="">
```
Comment (vivekmehto) [inline index.html]: here also

File: index.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                        <div class="box">
+                            <img src="https://images.unsplash.com/photo-1552422554-0d5af0c79fc6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
+                            <div class="content">
+                                <h1>Formal Shoes</h1>
+                                <p>Shop from a wide range of formal shoes.</p>
+                                <a href="#" class="btn">see More</a>
+                            </div>
+                        </div>
+                        <div class="box">
+                            <img src="https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
```
Comment (vivekmehto) [inline index.html]: give ``alt``

File: index.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+                        </div>
+                        <p class="text">
+                            ShoeStore contains a wide range of footwear products with multiple categories. SignUp to explore the store.
+                        </p>
+                        <div class="cta">
+                            <a href="#" class="btn">SignUp</a>
+                        </div>
+                    </div>
+                    <div class="right-side">
+                        <img src="./assets/heroImage.png" alt="" class="hero-image">
```
Comment (vivekmehto) [inline index.html]: provide ``alt`` to each and every image.

File: css/reset.css [hunk 5]
```diff
@@ -11,7 +11,7 @@ html{
 }
 
 body{
-    background-color: var(--white-color);
+    background-color: var(--white-color) !important;
```
Comment (rahulcs754) [inline css/reset.css:14]: try to avoid !important in css class
use CSS Specificity for apply into element

**Review by vivekmehto — COMMENTED** [review]
one suggestion add ``alt`` in every image. code looks good just try to keep CSS specificity low.

File: css/mediaQueries.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
+@media screen and (max-width: 870px){
```
Comment (rahulcs754) [inline css/mediaQueries.css:1]: you can use em unit in media query vikram sir told in lecture

**Review by rahulcs754 — APPROVED** [review]
All is good but some small suggestion that will help you in code.

File: css/reset.css (hunk 5 above)
Comment (RahulSingh9131) [reply css/reset.css:14]: Actually i have imported the styles from component library. so to overwrite the changes i have marked it with important.

File: css/mediaQueries.css (hunk 6 above)
Comment (RahulSingh9131) [reply css/mediaQueries.css:1]: yes i can , but to avoid calculations, i have used absolute units.

**Pull request merged**